monkestation/code/game/objects/items/anime.dm

```diff
@@ -137,6 +137,7 @@
 		weeb.add_splatter_floor(location)
 		var/msg = "<span class=danger>You feel the power of God and Anime flow through you!</span>"
 		to_chat(weeb, msg)
+		to_chat(weeb, "<span class=danger>Through the power of Anime, your food preferences now match your new form!</span>")
```

Comment:
Can we leave the God and Anime in there? It's an ancient meme reference

Reply:
Looks like it keeps it, but adds a second message on top.
Could probably just be combined into something like <span class=danger>You feel the power of God and Anime flow through you! Your tastes in food have also changed.</span>